### PR TITLE
fixing tests

### PR DESCRIPTION
**Pull request by aduewel**
Please squash merge after review, we should get rid of most of the meaningless commit messages :)

### CODE REVIEW

File: deploy/cloudformation_omaha.yml [hunk 1]
```diff
@@ -1,921 +1,921 @@
...
+    Default: cache.t2.micro
+    Description: InstanceClass for each Redis node.
+  SSHKeyPairName:
+    Type: String
+    Default: None
+    Description: EC2 SSH KeyPair Name
+  AppS3Bucket:
+    Type: String
+    Default: 'dentalwings-elasticbeanstalk-samples-us-west-1'
+    Description: BucketName where the Application Zip is located
```
Comment (joel-schaal) [inline deploy/cloudformation_omaha.yml:119]: Is that the good description ?

File: docker-compose.common.yml [hunk 2]
```diff
@@ -36,4 +36,4 @@ services:
       ELK_HOST: ''
       ELK_PORT: ''
       FILEBEAT_DESTINATION: ''
-      SENTRY_DSN: 'https://2b426b31861249db9e68d7ac755976b9@sentry.aws.dwos.com/3'
+      SENTRY_DSN: 'https://2b426b31861249db9e68d7ac755976b9:a562b533c3884fb5a0da8bc1b84af8f4@sentry.aws.dwos.com/3'
```
Comment (joel-schaal) [inline docker-compose.common.yml]: Can we inject that ?

File: deploy/cloudformation_omaha.yml (hunk 1 above)
Comment (aduewel) [reply deploy/cloudformation_omaha.yml:119]: any suggestion?

File: docker-compose.common.yml (hunk 2 above)
Comment (aduewel) [reply docker-compose.common.yml]: we should remove it ;)

File: omaha_server/omaha/tasks.py [hunk 3]
```diff
@@ -39,14 +37,11 @@
...
 from feedback.models import Feedback
-from sentry_sdk import capture_message
-
-logger = logging.getLogger(__name__)
 
 
 @app.task(ignore_result=True)
 def collect_statistics(request, ip=None):
-    statistics.collect_statistics(parse_request(request), ip=ip)
+    statistics.collect_statistics(parse_request(bytes(bytearray(request, encoding='utf-8'))), ip=ip)
```
Comment (aduewel) [inline omaha_server/omaha/tasks.py]: https://sentry.aws.dwos.com/dentalwings/omaha_server/issues/231/ something is still wrong here, the collect_statistics isn't able to get all requested data from the request :-(

**Review by jouvent — APPROVED** [review]
old things

**Pull request merged**